src/cron_tasks/monitoring_metrics.py

```diff
@@ -82,11 +82,13 @@ async def run_calculate_performance_metrics_pipeline(
         )
         return
 
+    labels = list(model.labels.values())
+
```

Comment:
This solution is going to be OK if those labels are given by the user. But what if they aren't provided since they're optional? What do you think @momegas ?

Reply:
Asked the same above in a different way @sinnec.....